packages/x-adapter-platform/src/schemas/responses/experience-controls-response.schema.ts

```diff
@@ -0,0 +1,11 @@
+import { createMutableSchema } from '@empathyco/x-adapter';
+import { ExperienceControlsResponse } from '@empathyco/x-types';
+import { PlatformExperienceControlsResponse } from '../../types';
+
+export const experienceControlsResponseSchema = createMutableSchema<
+  PlatformExperienceControlsResponse,
+  ExperienceControlsResponse
+>({
+  numberOfCarousels: 'numberOfCarousels',
+  resultsPerCarousels: 'numberOfCarousels'
+});
```

Comment:
This is the structure that a normal mapper for this endpoint would use, but there are 2 things to take into account:
1. The nature of the endpoint is open to change. Not only we don't know if these are the fields that the endpoint will return by default for all the setups, we also don't know if these are the only fields that it will return.
2. This structure is not compatible with the front components (the module). We need to have an object wrapping the configuration coming from the endpoint (`controls`, for example) because we'll also need an `events` object (populated in the setups, not here)


I believe something like this should work and makes sense for what we want to do (you'll need to adapt the typings):

```suggestion
>({
  controls: (response) => response,
  events: {}
});
```